addons/crm/tests/test_crm_lead.py

```diff
@@ -663,3 +663,39 @@ def test_phone_mobile_update(self):
         self.assertEqual(lead.phone, self.test_phone_data[1])
         self.assertEqual(lead.mobile, self.test_phone_data[2])
         self.assertFalse(lead.phone_sanitized)
+
+    @users('user_sales_manager')
+    def test_crm_lead_creation_partner_with_inactive_language(self):
+        """This test checks that creating a contact from a lead with an inactive language will ignore the language
+            while creating a contact from a lead with an active language will take it into account """
+        ResLang = self.env['res.lang'].sudo().with_context(active_test=False)
+
+        """Create a lead with an inactive language -> should ignore the preset language"""
+        lang_fr = ResLang.search([('code', '=', 'fr_FR')])
+        if not lang_fr:
+            lang_fr = ResLang._create_lang('fr_FR')
+        # set French language as inactive then try to call "_message_get_suggested_recipients"
+        # -> lang code should be ignored
+        lang_fr.active = False
+        lead1 = self.env['crm.lead'].create({
+            'name': 'TestLead',
+            'email_from': self.test_email,
+            'lang_id': lang_fr.id,
+        })
+        data = lead1._message_get_suggested_recipients()[lead1.id]
+        self.assertTrue(data[0][2] is None)
```

Comment:
While we are at it why not checking other values returned by message_get_suggested_recipients ? We have no tests for that I think, let us take the opportunity to cover a bit more code :)

Reply:
There is one here : https://github.com/odoo/odoo/blob/9bfebd99a7eeab6128a8ad8b19dad897e0b7e889/addons/test_mail/tests/test_mail_thread_internals.py#L203 , but if something else is needed to be tested I can of course add some more code in my test :)